stestr/selection.py

```diff
@@ -26,7 +27,17 @@ def filter_tests(filters, test_ids):
     """
     if filters is None:
         return test_ids
-    _filters = list(map(re.compile, filters))
+
+    _filters = []
+    for f in filters:
+        if isinstance(f, str):
+            try:
+                _filters.append(re.compile(f))
+            except re.error:
+                print("Invalid regex: %s provided in filters" % f)
```

Comment:
I was thinking like these error messages should be written to stderr. But it's a separate thing. We already do it in the others.
I'll push a patch to fix it if we should. What do you think?

Reply:
Yeah using stderr is probably the proper way to handle this. It's a pattern I've used in other places, mostly because I'm lazy. I think you pushing a follow up patch to switch all of those places to use stderr is a good idea.

Reply:
OK, cool. I'll push a patch for that later.